lib/validation/rules/checked.ts

```diff
@@ -1,9 +1,15 @@
 import { helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
 import { checked as baseChecked } from '../validators/checked'
 
+export const message = {
+  en: 'You must check the field.',
+  ja: 'この項目は選択が必須です。'
+}
+
 export function checked(msg?: string) {
-  return helpers.withMessage(
-    () => msg ?? 'You must check the field.',
-    (value: boolean) => !helpers.req(value) || baseChecked(value)
-  )
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang],
+    validation: (value: boolean) => !helpers.req(value) || baseChecked(value)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70